assets/src/edit-story/output/page.js

```diff
@@ -55,30 +58,32 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
       id={id}
       auto-advance-after={autoAdvance ? autoAdvanceAfter : undefined}
     >
-      {backgroundElements.length > 0 && (
-        <amp-story-grid-layer template="vertical">
-          <div className="page-background-area" style={backgroundStyles}>
-            {backgroundElements.map((element) => (
-              <OutputElement key={'el-' + element.id} element={element} />
-            ))}
-          </div>
-        </amp-story-grid-layer>
-      )}
-
-      {backgroundOverlay && backgroundOverlay !== OverlayType.NONE && (
-        <amp-story-grid-layer template="vertical">
-          <div
-            className="page-background-overlay-area"
-            style={{ ...backgroundOverlayStyles }}
-          />
+      {(backgroundElements.length > 0 || backgroundOverlayStyles) && (
```

Comment:
I don't think there's a case where you have `backgroundOverlayStyles` but no `backgroundElements` no?

Reply:
Not sure, but the old code seemed to have thought so. Can you confirm?